Life-Pill/pharmacy-pos-frontend
# Cashier temporary logout page done

# Pull Request

## Description
This pull request includes enhancements to the CashierTemporaryLogOutPage component and adds a smooth fade-out animation. Additionally, it introduces a new DateAndClock component and a RecentCashier component for better organization and reusability.

## Changes Made
- **CashierTemporaryLogOutPage:**
  - Added a fade-out animation using Tailwind CSS.
  - Updated the transition class for a smoother effect.

- **DateAndClock Component:**
  - Created a new component for displaying the date and clock.
  - Utilizes the react-live-clock library and moment.js for time formatting.

- **RecentCashier Component:**
  - Created a new component for displaying information about the recent cashier.
  - Includes an image, cashier details, and the name of the logged-in cashier.

## How to Test
1. Run the application.
2. Navigate to the CashierTemporaryLogOutPage.
3. Click on the page to trigger the fade-out animation.
4. Verify that the navigation to the login-cashier-password page occurs after the animation.

## Additional Notes
- Make sure to adjust the transition duration and easing function in the CashierTemporaryLogOutPage component according to your preferences.
- Feel free to customize the DateAndClock and RecentCashier components further based on your application's design.

## Checklist
- [x] Commits are descriptive.
